NachoSpigot-Server/pom.xml

```diff
@@ -35,7 +35,7 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-all</artifactId>
-            <version>4.1.68.Final</version>
+            <version>4.1.74.Final</version>
```

Comment:
Revert, some plugins don't work with 4.1.69+
```suggestion
            <version>4.1.68.Final</version>
```

Reply:
Can you let me know what's the issue on 4.1.69+?

Reply:
From https://netty.io/news/2021/10/11/4-1-69-Final.html

> This release changes the way how the netty-all artifact is packaged / produced. While before it did contain all the code of all the other modules directly with this release we changed to to just depend on all the other modules.

Reply:
Netty 4.1.69 is working with latest ProtocolLib dev build, just tested yesterday

Reply:
> Netty 4.1.69+ is working with latest ProtocolLib dev build, just tested yesterday

Yeah, since i provided the fix in a PR

Reply:
So I can revert it back to 4.1.74?

Reply:
It should be fine now since mainly only plugins that heavily modify/use NMS/Netty have issues and most of these plugins either updated to 4.1.69+ due to newer MC versions or don’t support 1.8 at all and wouldn’t work anyway.

Reply:
can we use diffent netty to load diffent plugins

Reply:
> can we use diffent netty to load diffent plugins

no